frontend/public/sitemap.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
 <sitemap><loc>https://www.asianweddingmakeup.com/sitemap-0.xml</loc></sitemap>
+<sitemap><loc>https://www.asianweddingmakeup.com/sitemap-1.xml</loc></sitemap>
```

Comment:
do we need both versions?

Reply:
it's not versions it's more like pages! Right now the sitemap config has 200 pages per file